engine/admin/logging/logentries.md

```diff
@@ -0,0 +1,51 @@
+---
+aliases: ["/engine/reference/logging/logentries/"]
```

Comment:
You don't need the brackets for aliases, but I don't know if it hurts anything to have them.